aws/elasticbeanstalk/environment/main.tf

```diff
@@ -98,7 +98,7 @@ resource "aws_elastic_beanstalk_environment" "env" {
   setting {
     namespace = "aws:autoscaling:asg"
     name      = "MaxSize"
-    value     = "2"
+    value     = "${var.environment_type == "SingleInstance" ? "1" : "2"}"
```

Comment:
I thought configuring the autoscaling with a `SingleInstance` made no sense, but I guess I was completely wrong

Reply:
I think it takes a default value, this 2 makes a always diff when generating the plan